package.json

```diff
@@ -36,13 +36,17 @@
     "@types/node": "^12.12.6",
     "@types/recursive-readdir": "^2.2.0",
     "@types/semver": "^6.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.16.1",
+    "@typescript-eslint/parser": "^4.16.1",
+    "eslint": "^7.21.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "jest": "^26.6.0",
     "prettier": "^2.2.1",
-    "rimraf": "^2.6.2",
-    "ts-jest": "^26.4.0",
-    "ts-node": "^8.2.0",
-    "tslint": "^5.17.0",
-    "typescript": "^4.1.0"
+    "rimraf": "^3.0.2",
+    "ts-jest": "^26.5.2",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.2"
```

Comment:
I guess these library updates were required for eslint?

Reply:
These last 4 weren't as far as I know. I could revert them if you want to?

Reply:
I'm ok with it.